scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3852,432 @@ def rvs(self, dim, size=1, random_state=None):
+        \sum_{i=1}^k x_i = n
+
+    where :math:`m_i` are the number of objects of type :math:`i`, :math:`M`
+    is the total number of objects in the population (sum of all
+    :math:`m_i`s), and :math:`n` is the size of the sample to be taken
+    from the population.
+
+    .. versionadded:: 1.6.0
+
+    Examples
+    --------
+
+    >>> from scipy.stats import multivariate_hypergeom
+    >>> rv = multivariate_hypergeom(m=[10, 20], n=12)
+    >>> rv.pmf(x=[8, 4])
+    0.0025207176631464523
+
+    The `multivariate_hypergeom` distribution is identical to the
+    corresponding `hypergeom` distribution (tiny numerical differences
+    notwithstanding) when only two types (good and bad) of objects
+    are present in the population:
+
+    >>> from scipy.stats import hypergeom
+    >>> multivariate_hypergeom.pmf(x=[3, 1], m=[10, 5], n=4)
+    0.4395604395604395
+    >>> hypergeom.pmf(k=3, M=15, n=4, N=10)
+    0.43956043956044005
+
+    The functions ``pmf``, ``logpmf``, ``mean``, ``var``, ``cov``, and ``rvs``
+    support broadcasting, under the convention that the vector parameters
+    (``x`` and ``m``) are interpreted as if each row along the last axis
+    is a single object. For instance:
+
+    >>> multivariate_hypergeom.pmf(x=[[3, 4], [4, 5]], m=[5, 10], n=[7, 9])
+    array([0.32634033, 0.25174825])
+
+    Here, ``x.shape == (2, 2)``, ``m.shape == (2,)``, and ``n.shape == (2,)``,
+    but following the rules mentioned above they behave as if the rows
+    ``[3, 4]`` and ``[3, 5]`` in ``x``, ``[5, 10]`` in ``m``, and ``[7, 9]``
+    in ``n`` were a single object, and as if we had ``x.shape = (2,)``,
+    ``m.shape = (2,)``, and ``n.shape = ()``. To obtain the individual
+    elements without broadcasting, we would do this:
+
+    >>> multivariate_hypergeom.pmf([3, 4], m=[5, 10], n=7)
+    0.3263403263403265
+    >>> multivariate_hypergeom.pmf([4, 5], m=[5, 10], n=9)
+    0.25174825174825205
+
+    This broadcasting also works for ``cov``, where the output objects are
+    square matrices of size ``m.shape[-1]``. For example:
+
+    >>> multivariate_hypergeom.cov(m=[[7, 9], [10, 15]], n=[8, 12])
+    array([[[ 1.05, -1.05],
+            [-1.05,  1.05]],
+           [[ 1.56, -1.56],
+            [-1.56,  1.56]]])
+
+    In this example, ``m.shape == (2, 2)`` and ``n.shape == (2,)``,
+    broadcast, following the rules above, as if ``m.shape == (2,)``.
+    Thus the result should also be of shape ``(2,)``, but since each output is
+    a :math:`2 \times 2` matrix. The result, in fact, has shape ``(2, 2, 2)``,
+    where ``result[0]`` is equal to
+    ``multivariate_hypergeom.cov(m=[7, 9], n=8)`` and ``result[1]`` is equal
+    to ``multivariate_hypergeom.cov(m=[10, 15], n=12)``.
+
+    See also
+    --------
+    scipy.stats.hypergeom : The Hypergeometric distribution.
+    numpy.random.Generator.multivariate_hypergeometric : The Multivariate Hypergeometric distribution sampler in numpy.
```

Comment:
```suggestion
    numpy.random.Generator.multivariate_hypergeometric : The Multivariate Hypergeometric distribution sampler in NumPy.
```

Reply:
fixed

Reply:
Almost, except for the stylization of NumPy. You can commit suggested changes directly, right? There should be a button "Add suggestion to batch", then when you've added the ones you want, you can commit.
![image](https://user-images.githubusercontent.com/6570539/93042509-11f82a80-f604-11ea-944e-0a850dbf9eb6.png)

Reply:
This is now fixed in the latest commit